share/commands/directive-replace

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+
+#. "${BASEDIR}/../lib/api_call.sh"
+. "/opt/rudder/share/lib/api_call.sh"
+
+TECHNIQUES_DIRECTORY="${CONFIGURATION_DIRECTORY}/technique"
+TOKEN=""
+DISPLAY_COMMAND=false
+API_URL="https://localhost/rudder"
```

Comment:
This one should go to api_call.sh
Ans you should prefer 127.0.0.1 instead of localhost for buggy /etc/hosts